src/Tribe/PUE/Checker.php

```diff
@@ -179,6 +179,20 @@ class Tribe__PUE__Checker {
 		 */
 		private $validate_query = [];
 
+		/**
+		 * A unique list of instances of the PUE Checker that has been initialized.
+		 *
+		 * @since TBD
+		 *
+		 * @var Tribe__PUE__Checker[] Instances of checkers that have been registered.
+		 */
+		protected static $instances = [];
+
+		/**
+		 * @var string The transient key.
+		 */
+		public const IS_ANY_LICENSE_VALID_TRANSIENT_KEY = 'IS_ANY_LICENSE_VALID_TRANSIENT';
```

Comment:
lets go with like `tec_...` to avoid a possible conflict.

Reply:
Whoops.